src/screens/OtherProfile/index.js

```diff
@@ -1042,39 +897,15 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     alignItems: 'center'
   },
-  toggleSwitchContainer: {display: 'flex', alignSelf: 'flex-end', paddingVertical: 10},
-  rightHeaderContentContainer: {display: 'flex', flexDirection: 'row'},
-  headerImageContainer: {display: 'flex', flexDirection: 'row', marginBottom: 20},
-  bioAndSendChatContainer: (dynamicColors) => ({
-    backgroundColor: dynamicColors.primary,
-    borderRadius: 15,
-    paddingHorizontal: 10,
-    paddingTop: 10
-  }),
-  bioText: (dynamicColors) => ({
-    color: dynamicColors.text,
-    fontSize: 14,
-    fontWeight: '600',
-    lineHeight: 22
-  })
+  rightHeaderContentContainer: {
+    display: 'flex',
+    flexDirection: 'row'
+  },
+  headerImageContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginBottom: 20
+  }
 });
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The PropTypes for `BioAndChat` component have been removed. If you're not using TypeScript or any other type checking tool, it's recommended to use PropTypes to validate the props being passed to the components.
```diff
- BioAndChat.propTypes = {
-   isAnonimity: PropTypes.bool,
-   bio: PropTypes.string,
-   openBio: PropTypes.bool,
-   isSignedMessageEnabled: PropTypes.bool,
-   showSignedMessageDisableToast: PropTypes.bool,
-   loadingGenerateAnon: PropTypes.bool,
-   avatarUrl: PropTypes.string,
-   anonProfile: PropTypes.object,
-   onSendDM: PropTypes.func,
-   setDMChat: PropTypes.func,
-   loadingSendDM: PropTypes.bool,
-   dmChat: PropTypes.string,
-   username: PropTypes.string,
-   toggleSwitch: PropTypes.func,
-   isAnonimityEnabled: PropTypes.bool
- };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->